indico/modules/categories/blueprint.py

```diff
@@ -45,6 +48,15 @@ def _redirect_event_creation(category_id, event_type):
 _bp.add_url_rule('/manage/move', 'move', RHMoveCategory, methods=('POST',))
 _bp.add_url_rule('/manage/protection', 'manage_protection', RHManageCategoryProtection, methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/settings', 'manage_settings', RHManageCategorySettings, methods=('POST', 'GET'))
+_bp.add_url_rule('/manage/roles', 'manage_roles', RHCategoryRoles, methods=('POST', 'GET'))
```

Comment:
I'd move this one down to "Role management" (and lowercase the m in that comment to be consistent with the others)